.vscode/launch.json

```diff
@@ -12,7 +12,8 @@
 			"request": "launch",
 			"args": [
 				"--extensionDevelopmentPath=${workspaceFolder}",
-				"--extensionDevelopmentKind=web"
+				"--extensionDevelopmentKind=web",
+				"C:\\Users\\riknoll\\OneDrive - Microsoft\\Documents\\vscode-test10"
```

Comment:
Is this something we could make generic?

Reply:
didn't mean to check that in...